frontend/src/libs/components/order-list-card/order-list-card-business.tsx

```diff
@@ -0,0 +1,83 @@
+import { type OrderResponseDto } from 'shared/build/index.js';
+
+import { IconName } from '~/libs/enums/icon-name.enum';
+import {
+  getValidClassNames,
+  jsonToLatLngLiteral,
+} from '~/libs/helpers/helpers.js';
+import { useCallback } from '~/libs/hooks/hooks.js';
+
+import { Badge, Icon } from '../components.js';
+import { statusConverter } from './libs/helpers/helpers.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  order: OrderResponseDto;
+  select: ({
+    startPoint,
+    endPoint,
+  }: {
+    startPoint: google.maps.LatLngLiteral;
+    endPoint: google.maps.LatLngLiteral;
+  }) => void;
+};
+
+const OrderListCardBusiness: React.FC<Properties> = ({
```

Comment:
I have done so, because we will also have a order list car for business. And I supposed to hold it inside order-list-card directory. But ok, I've changed.